### PR TITLE
Add SIG Scheduling spotlight

### PR DESCRIPTION
**Pull request by fsmunoz**
Add SIG Scheduling spotlight interview.

Closes: #519 #508

### CODE REVIEW

File: content/en/blog/2024/sig-scheduling-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+[slack.k8s.io](https://slack.k8s.io), and after that join the `#k8s-code-organization` channel. There is also a
+[code-organization office
+hours](https://github.com/kubernetes/community/tree/master/sig-architecture#meetings) that takes
+place that you can choose to attend. Timezones are hard, so feel free to also look at the recordings
+or meeting notes and follow up on slack!
+
+**FSM**: Excellent, thank you! Any final comments you would like to share?
+
+**MJ**: The Code Organization subproject always needs help! Especially areas like the publishing
+bot, so don’t hesitate to get involved in the `#k8s-code-organization` Slack channel.
```
Comment (ArvindParekh) [inline content/en/blog/2024/sig-scheduling-spotlight.md:384]:
```suggestion
```

This is the sig-architecture spotlight blog content. Maybe we should remove this. Or is this intentional?

**Review by ArvindParekh — COMMENTED** [review]
Thanks for the PR, Frederico! 

Looks good, except for the part where it introduces the SIG-Architecture spotlight as well. I'm guessing it's a copy-paste mix-up and not intentional 😄 
We should probably remove that part.

**Comment by fsmunoz** [comment]
@ArvindParekh indeed! Removed it :)

**Comment by sftim** [comment]
/hold

I assume we want to mirror this to the [main blog](https://blog.k8s.io/); if that's right, let's hold this until both PRs are completely ready to merge.

**Comment by fsmunoz** [comment]
@sftim https://github.com/kubernetes/website/pull/47731 opened.

**Comment by fsmunoz** [comment]
Updated publication date for next week in both this and the website PR.

File: content/en/blog/2024/sig-scheduling-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+---
+layout: blog
+title: "Spotlight on SIG Scheduling"
+slug: sig-scheduling-spotlight-2024
+date: 2024-09-17
```
Comment (fsmunoz) [inline content/en/blog/2024/sig-scheduling-spotlight.md:5]:
```suggestion
date: 2024-09-24
```

**Comment by fsmunoz** [comment]
Updated publication date for next week. I think this is good to go @sftim

**Comment by sftim** [comment]
This is ready to be reviewed.

**Comment by sftim** [comment]
I've been hoping this would get a review from someone else, because having just one active reviewer scales poorly. Anyway:

/lgtm
/approve

File: content/en/blog/2024/sig-scheduling-spotlight.md
[the remark this answers is not in the document]
Comment (fsmunoz) [reply content/en/blog/2024/sig-scheduling-spotlight.md:1]: Oh, I usually review that, missed it this time. I'll submit a PR tomorrow, thanks!

**Pull request merged**